src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -0,0 +1,1227 @@
+                .flags(FLAMMABLE, NO_SMELTING, NO_SMASHING, MORTAR_GRINDABLE)
+                .components(Carbon, 1)
+                .build();
+
+        Chromite = new Material.Builder(267, "chromite")
+                .dust(1).ore()
+                .color(0x23140F).iconSet(METALLIC)
+                .components(Iron, 1, Chrome, 2, Oxygen, 4)
+                .build();
+
+        Cinnabar = new Material.Builder(268, "cinnabar")
+                .gem(1).ore()
+                .color(0x960000).iconSet(EMERALD)
+                .flags(CRYSTALLIZABLE, DECOMPOSITION_BY_CENTRIFUGING)
+                .components(Mercury, 1, Sulfur, 1)
+                .build();
+
+        Water = new Material.Builder(269, "water")
+                .fluid()
+                .color(0x0000FF)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Hydrogen, 2, Oxygen, 1)
+                .build();
+
+        Clay = new Material.Builder(270, "clay")
+                .dust(1)
+                .color(0xC8C8DC).iconSet(ROUGH)
+                .flags(MORTAR_GRINDABLE)
+                .components(Sodium, 2, Lithium, 1, Aluminium, 2, Silicon, 2, Water, 6)
+                .build();
+
+        Coal = new Material.Builder(271, "coal")
+                .gem(1, 1600).ore() //default coal burn time in vanilla
+                .color(0x464646).iconSet(LIGNITE)
+                .flags(FLAMMABLE, NO_SMELTING, NO_SMASHING, MORTAR_GRINDABLE, EXCLUDE_BLOCK_CRAFTING_BY_HAND_RECIPES, DISABLE_DECOMPOSITION)
+                .components(Carbon, 1)
+                .build();
+
+        Cobaltite = new Material.Builder(272, "cobaltite")
+                .dust(1).ore()
+                .color(0x5050FA).iconSet(METALLIC)
+                .components(Cobalt, 1, Arsenic, 1, Sulfur, 1)
+                .build();
+
+        Cooperite = new Material.Builder(273, "cooperite")
+                .dust(1).ore()
+                .color(0xFFFFC8).iconSet(METALLIC)
+                .components(Platinum, 3, Nickel, 1, Sulfur, 1, Palladium, 1)
+                .build();
+
+        Cupronickel = new Material.Builder(274, "cupronickel")
+                .ingot(1).fluid()
+                .color(0xE39680).iconSet(METALLIC)
+                .flags(EXT_METAL, GENERATE_SPRING)
+                .components(Copper, 1, Nickel, 1)
+                .itemPipeProperties(2048, 1)
+                .cableProperties(GTValues.V[1], 1, 1)
+                .build();
+
+        DarkAsh = new Material.Builder(275, "dark_ash")
+                .dust(1)
+                .color(0x323232)
+                .flags(DISABLE_DECOMPOSITION)
+                .components(Carbon, 1)
+                .build();
+
+        Diamond = new Material.Builder(276, "diamond")
+                .gem(3).ore()
+                .color(0xC8FFFF).iconSet(DIAMOND)
+                .flags(GENERATE_BOLT_SCREW, GENERATE_LENS, GENERATE_GEAR, NO_SMASHING, NO_SMELTING, FLAMMABLE, GENERATE_BOLT_SCREW,
```

Comment:
Since I am storing flags in a Set now, there isn't any way for duplicate flags to ever actually happen, but yeah this shouldn't be registered like this